CONTRIBUTING.md

```diff
@@ -19,8 +19,9 @@ our own CI/CD: the ["superchain" image][superchain] from.
 The image can be built for local usage, too:
 
 ```console
+$ cd jsii # go to the root of the jsii repo
 $ IMAGE=superchain
-$ docker build -t ${IMAGE} ./superchain
+$ docker buildx build -t $IMAGE --target $IMAGE -f superchain/Dockerfile .
```

Comment:
```suggestion
$ docker build -t ${IMAGE} -f superchain/Dockerfile .
```

Thanks for finding this! Let's not confuse people with `buildx` just yet though.